docs/content/practices/test_runners_review.md

```diff
@@ -72,25 +75,25 @@ do the whole job for you.
 ➕ Dynamic test batching (test count/test duration) <br>
 ➕ Smart retries with a quotas <br>
 ➕ Screenshots & video out of the box <br>
-➕ Improved test report with video & logs <br>
-➕ Automatically rebalanced test execution if connecting/disconnecting devices on fly <br>
+➕ Improved test report with video & logs <br> 
+➕ Automatically rebalanced test execution if connecting/disconnecting devices on the fly <br>
 ➕ Pull files from the device after test run,
 e.g. [allure-kotlin](https://github.com/allure-framework/allure-kotlin) <br>
-➕ Basic [Allure](https://github.com/allure-framework) support out of the box <br>
+➕ Basic [Allure](https://github.com/allure-framework) support out of the box <br> 
```

Comment:
It is not clear what Allure does for someone who has not used it:
video (what is mentioned above), reports...
I'd rather mention the most relevant, that have not been mentioned above, or exclude video form above and put it here (can Marathon record videos without Allure?) @NoNews